public/v2/apps/phpldapadmin.json

```diff
@@ -0,0 +1,77 @@
+{
+    "captainVersion": "2",
+    "documentation": "https://github.com/osixia/docker-openldap",
+    "description": "OpenLDAP with administration interface",
+    "displayName": "OpenLDAP + phpLDAPadmin",
+    "dockerCompose": {
+        "version": "3.8",
+        "services": {
+            "$$cap_appname": {
+                "image": "osixia/openldap:$$cap_openldap_version",
+                "restart": "always",
+                "ports": ["389:389", "636:636"],
+                "environment": {
+                    "LDAP_ORGANISATION": "$$cap_openldap_ldap-organisation",
+                    "LDAP_DOMAIN": "$$cap_openldap_ldap-domain",
+                    "LDAP_ADMIN_PASSWORD": "$$cap_openldap_ldap-admin-password"
+                }
+            },
+            "php$$cap_appnameadmin": {
+                "image": "osixia/phpldapadmin:$$cap_phpldapadmin_version",
+                "containerHttpPort": "443",
+                "restart": "always",
+                "depends_on": ["$$cap_appname"],
+                "environment": {
+                    "PHPLDAPADMIN_TRUST_PROXY_SSL": true,
+                    "PHPLDAPADMIN_LDAP_HOSTS": "$$cap_appname.$$cap_root_domain",
+                    "PHPLDAPADMIN_SERVER_ADMIN": "$$cap_phpldapadmin_admin-email",
+                    "PHPLDAPADMIN_SERVER_PATH": "https://php$$cap_appnameadmin.$$cap_root_domain"
+                }
+            }
+        }
+    },
+    "instructions": {
+        "start": "OpenLDAP Software is an open source implementation of the Lightweight Directory Access Protocol.",
+        "end": "OpenLDAP is deployed and available as srv-captain--$$cap_appname:389 to other apps."
+    },
+    "variables": [
+        {
+            "id": "$$cap_openldap_version",
+            "label": "Openldap Version",
+            "defaultValue": "1.4.0",
+            "description": "Checkout their docker page for the valid tags https://hub.docker.com/r/osixia/openldap/tags",
+            "validRegex": "/^([^\\s^\\/])+$/"
+        },
+        {
+            "id": "$$cap_openldap_ldap-organisation",
+            "label": "Ldap Organisation",
+            "description": "The Organization name",
+            "validRegex": "/.{1,}/"
+        },
+        {
+            "id": "$$cap_openldap_ldap-domain",
+            "label": "Ldap Host",
+            "description": "The host",
```

Comment:
Maybe provide an example in the description as the regex rule is quite complex to understand. Something like `myhost.domain.something` would work, right?

Reply:
Ok I'll do that! I take the advice into accounts for the next PRs :smile: